lib/attributor/attribute.rb

```diff
@@ -137,8 +137,7 @@ def example_from_options(parent, context)
       when ::Regexp
         val.gen
       when ::Array
-        # TODO: handle arrays of non native types, i.e. arrays of regexps.... ?
-        val.pick
+        val
```

Comment:
So for a collection, I suppose you'd have to do `[[1,2,3]]` to get a single example array of `[1,2,3]`